README.md

```diff
@@ -60,6 +66,17 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### Before making a Pull request
+
+Make sure to run the following commands:
+
+- `pnpm validate-scenarios`
```

Comment:
should we add this directly to the template?

Reply:
well the CI will complain if you don't do it, so feels like an extra noise in the template